### PR TITLE
Add hotel information

### PR DESCRIPTION
**Pull request by jessealama**
How does this look @sorawee ?

### CODE REVIEW

**Comment by sorawee** [comment]
For hotel information, I would suggest not putting this out for now. I'll try to contact a hotel that we have in mind for the block by the end of today. Will email you the result.

I think https://www.cs.washington.edu/visitors/getting_here#hotels is a nice resource, but let's also add a parenthetical remark that "Hotel Deca" is renamed to "[Graduate Hotels](https://graduatehotels.com/seattle/)".

It looks like the commit to add Jack's talk shouldn't be in this PR?

**Comment by jessealama** [comment]
The commit for @jackfirth 's talk accidentally snuck in. I just removed it.

File: rcon/2024/index.rkt [hunk 1]
```diff
@@ -408,8 +408,9 @@ Please come with your big questions and discussion topics.
...
- )
+   @paragraph{No official hotel has been selected, and no block(s) of rooms have been reserved. However, the University of Washington has @(a #:href "https://www.cs.washington.edu/visitors/getting_here#hotels" #:title "Accomodations near the Paul G. Allen School of Computer Science & Engineering" "a useful list") of nearby hotel recommendations on its Getting Here page. @(strong "NB") On that list of recommended hotels, Hotel Deca has been renamed to @(a #:href "https://graduatehotels.com/seattle/" #:title "Graduate Hotel" "Graduate Hotel").}
+    @paragraph{There will be a @(#:href "https://www.espn.com/college-football/game/_/gameId/401628505/michigan-washington" #:title "Washington vs. Michigan" "college football game") on October 5th, so there might not be parking space for those bringing their own car. It should also be fine if to say somewhere in the downtown area somewhat away from the university, though public transportation will likely be crowded because of the game.}
```
Comment (sorawee) [inline rcon/2024/index.rkt:412]: say -> stay?

Comment (jessealama) [reply rcon/2024/index.rkt]: Good spot -- fixed!

File: rcon/2024/index.rkt [hunk 2]
```diff
@@ -409,7 +409,7 @@ Please come with your big questions and discussion topics.
...
-    @paragraph{There will be a @(#:href "https://www.espn.com/college-football/game/_/gameId/401628505/michigan-washington" #:title "Washington vs. Michigan" "college football game") on October 5th, so there might not be parking space for those bringing their own car. It should also be fine if to stay somewhere in the downtown area somewhat away from the university, though public transportation will likely be crowded because of the game.}
+    @paragraph{There will be a @(a #:href "https://www.espn.com/college-football/game/_/gameId/401628505/michigan-washington" #:title "Washington vs. Michigan" "college football game") on October 5th, so there might not be parking space for those bringing their own car. It should also be fine if to stay somewhere in the downtown area somewhat away from the university, though public transportation will likely be crowded because of the game.}
```
Comment (sorawee) [inline rcon/2024/index.rkt:412]: Sorry, another spot "if to stay" doesn't sound right. Maybe just "to stay"?

Comment (jessealama) [reply rcon/2024/index.rkt]: fixed!

**Comment by jessealama** [comment]
What do you think @sorawee ?

**Comment by jessealama** [comment]
I'm merging this in so that people can have some information about hotels fairly early. We can always update this if we manage to get a special rate.

**Comment by sorawee** [comment]
Oh yeah, sorry! I somehow thought I already replied that it looks good to me... Clearly I didn't do that.

**Pull request merged**